kernel/crates/kernel_abi/src/mman.rs

```diff
@@ -0,0 +1,12 @@
+// Memory protection flags
+pub const PROT_NONE: i32 = 0x0;
+pub const PROT_READ: i32 = 0x1;
+pub const PROT_WRITE: i32 = 0x2;
+pub const PROT_EXEC: i32 = 0x4;
+
+// Memory mapping flags
+pub const MAP_SHARED: i32 = 0x01;
+pub const MAP_PRIVATE: i32 = 0x02;
+pub const MAP_FIXED: i32 = 0x10;
+pub const MAP_ANONYMOUS: i32 = 0x20;
+pub const MAP_ANON: i32 = MAP_ANONYMOUS;
```

Comment:
Please use the bitflags crate and macro to make this more rusty, like `Prot::None` or `MapFlags::Private` (or other names if you have better ones).